### ISSUE TITLE
Decoding peer endpoint produces unpack_from buffer length error

### ISSUE THREAD

**Issue by tateg** [ArgosyLabs/wgnlpy#4]
When I attempt to get a list of peers for a particular interface, if any of those peers have an endpoint then an error is produced on the decode step.

The error produced is:
```
decoding WGPEER_A_ENDPOINT
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pr2modules/netlink/__init__.py", line 1507, in try_to_decode
    cell.decode()
  File "/home/admin/wgnlpy/wgnlpy/nlas/sockaddr.py", line 24, in decode
    family, = unpack_from("H", self["value"])
struct.error: unpack_from requires a buffer of at least 2 bytes
```

This is an example peer that reproduces the error:
```
peer: <PUBKEY>
  endpoint: 24.6.228.60:49565
  allowed ips: 10.99.99.98/32, fe80:8a10::16/128
  transfer: 0 B received, 17.20 KiB sent
  persistent keepalive: every 30 seconds
```

The peers list is still returned by the module but the endpoint is a byte value:
```
'endpoint': b'\x14\x00\x04\x00\x02\x00\xc1\x9d\x18\x06\xe4<\x00\x00\x00\x00\x00\x00\x00\x00'
```

I've tested this on a few different versions of WireGuard and the result is the same:
- v1.0.20200513
- v1.0.20210223

It doesn't break the overall functionality of the module (unless you care about the peer endpoints), but does produce annoying stderr output when iterating over peers, etc.

**Comment by liske** [comment]
The value of self in [line 24 of `sockaddr.py`](https://github.com/ArgosyLabs/wgnlpy/blob/v0.1.4/wgnlpy/nlas/sockaddr.py#L24) is:

```
{'value': b'\x02', 'header': {'length': 20, 'type': 4}}
```

But `value` should be 20 bytes long, shouldn't it?

**Comment by pallas** [comment]
What versions of pyroute2 are you using? Admittedly, I haven't updated either pyroute2 or wireguard on my system in a while but I'm happy to check with newer versions.

**Comment by tateg** [comment]
I'm using pyroute2 v0.6.4

**Comment by pallas** [comment]
Ok, so going from 0.5.12 to 0.6.4 regresses here. I'll take a look at what changed.

**Comment by pallas** [comment]
I understand the issue & have a fix. The special case of decode unpacking "s" used to unpack the entire byte array but now it only unpacks the first byte. This doesn't happen for encode because "s" knows the length anyway. The reason it matters here is because we might get different sized sockaddr structs for different protocol families. I added code to just extract the entire NLA as bytes.

**Comment by pallas** [comment]
Fixed in v0.1.5

**Comment by liske** [comment]
Thanks!

(tested on pyroute2 0.5.19)
